TranslatorReasonerAPI.yaml

```
@@ -422,3 +449,199 @@ components:
       description: A Compact URI
       externalDocs:
         url: https://www.w3.org/TR/2010/NOTE-curie-20101216/
+    KnowledgeMap:
+      type: object
+      properties:
+        nodes:
+          type: object
+          description: >-
+            List of semantic types provided by this knowledge source.
+          additionalProperties:
+            $ref: '#/components/schemas/KMNode'
+        predicates:
+          type: array
+          description: >-
+            List of predicates provided by this knowledge source.
+          items:
+            $ref: '#/components/schemas/KMPredicate'
+      description: >-
+        Knowledge-map representation of this knowledge source.
+    KMNode:
+      type: object
+      description: >-
+        Description of semantic types provided by this knowledge source.
+      properties:
+        id_prefixes:
+          type: array
+          description: >-
+            List of CURIE prefixes that this knowledge source understands and
+            accepts on the input.
+          items: 
+            type: string
+          example: ['ChEMBL:','inchikey:']
+        count:
+          type: integer
+          description: >-
+            Total number of node instances known to this knowledge source,
+            -1 if the number is not known
+        count_by_source:
+          type: object
+          description: >-
+            Number of node instances by their source
+          additionalProperties:
+            type: integer
+          example:
+            ChEMBL: 1234
+        attributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KMAttribute'
+          description: >-
+            Attributes associated with this node.
+      required:
+        - id_prefixes
+      additionalProperties: false
+    KMPredicate:
```

```suggestion
    KMEdge:
```
The "predicate" is a fairly well-defined string property of this meta-graph edge.

---

This proposal is accepted